packages/prisma/migrations/20230622164946_remove_host_remnants_from_old_team_members/migration.sql

```diff
@@ -0,0 +1,16 @@
+
+DELETE FROM "Host" 
```

Comment:
In words; Find all hosts that no longer have memberships because they have since been deleted from the team, and remove these hosts.